CMakeLists.txt

```diff
@@ -431,7 +431,11 @@ while(library_variants_copy)
         set(target "aarch64-none-elf")
         set(cpu_family aarch64)
     else()
-        set(target "${target_arch}-none-eabi")
+        if(target_arch STREQUAL "armv6m")
```

Comment:
A brief comment here please explaining why this logic is needed

Reply:
Done (although I'm not that good at 'brief')